https://trac.macports.org/ticket/52358

It is an error to use "defined" with Perl 5.22 and later.

Upstream-Status: Inappropriate [no upstream]

Index: fixpaths
===================================================================
--- fixpaths.orig
+++ fixpaths
@@ -7,7 +7,7 @@ $usage = "Usage: $0 [-x<file dot-suffix>
 
 $ext="out";
 
-if (!defined(@ARGV)) { die ("$usage"); }
+if (!@ARGV) { die ("$usage"); }
 
 # read in the command line and get some definitions
 while ($_=$ARGV[0], /^-/) {
@@ -26,7 +26,7 @@ while ($_=$ARGV[0], /^-/) {
   }
 } # while parsing arguments
 
-if (!defined(%def)) {
+if (!%def) {
   die ("$0: nothing to do - no substitutions listed!\n");
 }
 
